app/Resources/views/autoEdits/result.html.twig

```diff
@@ -0,0 +1,84 @@
+{% extends 'base.html.twig' %}
+{% import 'macros/layout.html.twig' as layout %}
+
+{% block body %}
+    <div class="col-lg-12">
```

Comment:
this shouldn't be needed either. Just `<div class="panel panel-primary">` which frankly might could be moved to base.html.twig as well, but save that for another day